apps/src/sites/studio/pages/devise/registrations/edit.js

```diff
@@ -37,6 +44,9 @@ $(document).ready(() => {
       manageLinkedAccountsMountPoint,
       userType,
       authenticationOptions,
+      isPasswordRequired,
```

Comment:
going to rename `isPasswordRequired` to `userHasPassword` in a follow-up PR now that it has more than one use. (was previously only used in [ChangeEmailForm](https://github.com/code-dot-org/code-dot-org/blob/368a5fe379bbcb8c252566dce8110d05788f1c5a/apps/src/lib/ui/accounts/ChangeEmailForm.jsx#L93-L111))